data/simpy.gram

```diff
@@ -53,7 +53,9 @@ dotted_as_names[asdl_seq*]: a=dotted_as_name b=(',' c=dotted_as_name { c })* { s
 dotted_as_name[alias_ty]:
     | a=dotted_name 'as' b=NAME { _Py_alias(((expr_ty) a)->v.Name.id, ((expr_ty) b)->v.Name.id, p->arena) }
     | a=dotted_name { _Py_alias(((expr_ty) a)->v.Name.id, NULL, p->arena) }
-dotted_name[expr_ty]: a=NAME b=('.' c=NAME { c })* { seq_to_dotted_name(p, seq_insert_in_front(p, a, b)) }
+dotted_name[expr_ty]:
+    | a=NAME b=('.' c=dotted_name { c }) { join_names_with_dot(p, a, b) }
```

Comment:
This can be simplified to `a=NAME '.' b=dotted_name { join_names_with_dot(p, a, b) }`, i.e. without the parentheses.

Also it would seem to better match the semantics to make it left-recursive. (But right-recursive is probably more efficient.)